packages/core/src/app/ui/autocomplete/Autocomplete.tsx

```diff
@@ -33,7 +35,7 @@ class Autocomplete extends PureComponent<AutocompleteProps> {
 
         return (
             <Downshift
-                defaultHighlightedIndex={0}
+                defaultHighlightedIndex={defaultHighlightedIndex}
```

Comment:
Its default value is `null` according to the `Downshift` [doc](https://github.com/downshift-js/downshift#defaulthighlightedindex), hence I didn't set another default value here.